test/integration/narrow/routes/capture-debt.test.js

```diff
@@ -141,7 +141,7 @@ describe('capture-debt route', () => {
     }
 
     const result = await server.inject(options)
-    expect(result.request.response.source.context.model.errorSummary[0].text).toEqual('The scheme must be one of the following: SFI, SFI Pilot, LNR, Vet Visits, Lump Sums.')
+    expect(result.request.response.source.context.model.errorSummary[0].text).toEqual('The scheme must be one of the following: SFI, SFI Pilot, CS, BPS, FDMR, Vet Visits, Lump Sums.')
```

Comment:
See this comment: https://github.com/DEFRA/ffc-pay-request-editor/pull/69/files#r1145027610